ui/components/app/wallet-overview/btc-overview.tsx

```diff
@@ -20,8 +22,12 @@ const BtcOverview = ({ className }: BtcOverviewProps) => {
   const isBtcBuyable = useSelector(getIsBitcoinBuyable);
   ///: END:ONLY_INCLUDE_IF
 
+  // FIXME: This causes re-renders, so use isEqual to avoid this
```

Comment:
This comment gives the impression that the current code is the one causing the re-renders.

Reply:
I actually re-used some changes coming from the mainline, and there's no mention to `isEqual` so I removed that entirely!